### PR TITLE
[GECO-49] send exceptions to september

### PR DESCRIPTION
**Pull request by snilapwa**

### CODE REVIEW

File: cassiopeia/src/main/java/edu/asu/diging/gilesecosystem/cassiopeia/web/EditPropertiesController.java [hunk 1]
```diff
@@ -83,6 +88,7 @@ public String storeSystemConfig(@Validated @ModelAttribute SystemConfigPage syst
             model.addAttribute("show_alert", true);
             model.addAttribute("alert_type", "danger");
             model.addAttribute("alert_msg", "An unexpected error occurred. System Configuration could not be saved.");
+            messageHandler.handleMessage("An unexpected error occurred. System Configuration could not be saved.", e, MessageType.ERROR);
```
Comment (jdamerow) [inline cassiopeia/src/main/java/edu/asu/diging/gilesecosystem/cassiopeia/web/EditPropertiesController.java]: This message should be more specific about what happened and not the message for the users.

File: cassiopeia/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java [hunk 2]
```diff
@@ -327,7 +335,7 @@ public void run() {
                     for (int n = reader.read(buffer); n != -1; n = reader.read(buffer))
                         out.append(buffer, 0, n);
                 } catch (IOException e) {
-
+                    messageHandler.handleMessage("IOException", e, MessageType.ERROR);
```
Comment (jdamerow) [inline cassiopeia/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java]: let's be more helpful here. at say when this happened and in what situation.

File: cassiopeia/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java [hunk 3]
```diff
@@ -275,7 +283,7 @@ public Integer call() throws Exception {
 
         } catch (ExecutionException e) {
             // should not be thrown
-
+            messageHandler.handleMessage("Exceution Exception", e, MessageType.ERROR);
```
Comment (jdamerow) [inline cassiopeia/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java]: same here. be more specific what happened.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by snilapwa** [comment]
@jdamerow updated as per review comments

**Pull request merged**